__tests__/cmds/openapi.test.js

```diff
@@ -68,36 +68,7 @@ describe('rdme openapi', () => {
     });
   });
 
-  it('should error if API errors', () => {
```

Comment:
There's a whole bunch of nits like this in this changeset but the gist of what I did in this file are:

* Move errors into an "error handling" describe.
* Fix some confusing `it` language.
* Add some tests to make sure that errors that are coming back from `oas-normalize` (and `openapi-parser`) are of the new format.